scanner.cs

```diff
@@ -3,66 +3,176 @@
 using System.Collections.Generic;
 
 public class Scanner {
-	private int curByte = 0;
-	private byte[] __bytes;
-	private List<Tuple<string, TOKENS>> __tokens;
+    private int curByte = 0;
+    private int column = 1;
+    private int line = 1;
```

Comment:
Also, I'm going along with the dunders for the duration of the project, but I want to go on record as saying I hate dunders. They look ugly. In python at least they mean something, but here I don't believe they do.